services/api-server/src/simcore_service_api_server/api/dependencies/rabbitmq.py

```diff
@@ -39,6 +39,20 @@ async def wait_till_log_distributor_ready(app) -> None:
     return
 
 
+@retry(
+    wait=wait_fixed(2),
+    stop=stop_after_delay(_MAX_WAIT_FOR_LOG_DISTRIBUTOR_SECONDS),
+    before_sleep=before_sleep_log(_logger, logging.WARNING),
+    reraise=True,
+)
+async def wait_till_rabbitmq_ready(app) -> None:
```

Comment:
Question: there is the very same function (or at least exact same name) in https://github.com/ITISFoundation/osparc-simcore/blob/ecfdca7ee58f89222941189d2244edb6c22fbb03/packages/service-library/src/servicelib/rabbitmq/_utils.py, why is this one necessary?

Reply:
It is not. In fact it seems I didn't even use it anywhere. I have removed it. Thanks